CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## [1.13.0] - 2019-04-01
+### Added
+ * Add MaxCloseTime as a graceful close timeout (#724)
+
+### Changed
+ * Eliminate []byte(string) allocation under thrift transport (#729)
+ * Use atomic.Bool from go.uber.org/atomic v1.3 (#726)
+
+### Fixed
+ * Check max string length when using WriteLenXString (#728)
```

Comment:
More user-friendly: Return an error if transport header keys/values exceed the maximum allowed string length.

(that's the only case affected by this change iirc)